RedHat-UX/red-hat-design-system
# docs(cta): update docs

Closes #905
Closes #915 

## What I did

1. Updated the docs for rh-cta element based on the specs mentioned in https://github.com/RedHat-UX/red-hat-design-system/issues/905 



## Testing Instructions
1. Review the updated docs for [rh-cta(preview)](https://deploy-preview-1045--red-hat-design-system.netlify.app/elements/call-to-action/) and check if they match the [design spec](https://xd.adobe.com/view/6f3078cf-c5d5-4c7b-8e30-509acd11525c-3a63/) and the [content](https://docs.google.com/document/d/15pIDwmcCXYweZAcBx9ImnZ0DAlf-KAqHu98QRrtAJe0/edit#)

## Notes to Reviewers
